api_tests/providers/preprints/views/test_preprint_provider_subjects_list.py

```diff
@@ -83,6 +83,14 @@ def subj_a(self, provider):
     def subj_aa(self, provider, subj_a):
         return SubjectFactory(provider=provider, text='AA', parent=subj_a, highlighted=True)
 
+    @pytest.fixture()
+    def other_subj(self, provider):
```

Comment:
very minor; subj_aa is <thing it is>_<info about thing> and these are reversed. My recollection from python style guide is the `subj_other` form but feel free to leave as is.